.envrc

```diff
@@ -121,16 +121,17 @@ make setup-git-config
 
 ### Python ###
 
-# we're enforcing that virtualenv be in .venv{,3}, since future tooling e.g.
-# venv-update will rely on this.
 venv_name=".venv"
-python_version="python2.7"
+python_version="3.6"
 
-if [ "$SENTRY_PYTHON3" = "1" ]; then
-    venv_name=".venv3"
-    python_version="python3.6"
+# This is to support developer still use .venv with Python 2
+if [[ "$(python -V 2>&1)" = "Python $(grep "2.7" .python-version)" ]]; then
```

Comment:
This doesn't make sense. We need to check if `.venv` contains Python 2. It hasn't been activated at this point. Just remove this check. This is also why I moved this to ensure-venv.sh, aside from colorization. What you had before was fine:

```
if [ "$SENTRY_PYTHON2" = "1" ]; then
    venv_name=".venv2"
    python_version="2.7"
fi
```

Reply:
@joshuarli It is needed to support the usage of `.venv` w/o requiring the user to manually have to set `SENTRY_PYTHON2`.

At this point, I would like to stop making more tweaks since this code can so easily be broken.

Reply:
The replacement I suggested _is needed_. I'm pointing out that the python version check you have is wrong. And this can break if python resolves to python3, so it's a bug that needs to be fixed, not a tweak.

Reply:
I've tested it for all case scenarios and it does not fail.

Please let me know what your test case or how you get to that conclusion. I am happy to learn.

Reply:
Just because it works for all your test cases doesn't mean it's not a bug. I've already written down everything. Here it is again: the python check you have there is _before_ the venv is activated. So if it resolves to like, Python 3, the entire thing will be skipped and SENTRY_PYTHON2 doesn't do anything.

We want SENTRY_PYTHON2 to work in all cases. 

<img width="647" alt="image" src="https://user-images.githubusercontent.com/14209781/101073104-e0ee1d00-3596-11eb-81f0-14eae930e8f5.png">

Reply:
Thank you Josh! I was not testing that test case.